docs/content/bronnen/overzicht.md

```diff
@@ -0,0 +1,26 @@
+# Overzicht relevante bronnen
+
+## Algemeen
+- [Metamodel
+voor informatiemodellen](https://docs.geostandaarden.nl/mim/def-st-mim10-20170614/doc.pdf) v1.0
+- [API en URI strategie DSO](https://aandeslagmetdeomgevingswet.nl/digitaal-stelsel/documenten/documenten/api-uri-strategie/)
+- [Open Standaarden Forum Standaardisatie](https://www.forumstandaardisatie.nl/open-standaarden)
+
+## GEMMA 2
+- [GEMMA 2](https://www.gemmaonline.nl/index.php/GEMMA_Architectuur)
+
+
+## Zaakgericht Werken
+- [GEMMA 2 Katern Zaakgericht Werken](https://www.gemmaonline.nl/index.php/GEMMA_2_Katern_Zaakgericht_Werken)
+- [Zaakgericht Werken in GEMMA 2](https://www.gemmaonline.nl/index.php/ZGW_in_GEMMA_2)
+
+
+## RGBZ 2
+- [Informatiemodel RGBZ](https://www.gemmaonline.nl/index.php/Informatiemodel_RGBZ_2.0.1_-_specificaties_-_CONCEPT_20180605) (v2.0.1 - concept)(.zip!)
+- [Klantcontacten en het RGBZ](https://www.gemmaonline.nl/index.php/Klantcontacten_en_het_RGBZ)
+
+
+## Informatiebeheer
+- [Selectielijst gemeenten en intergemeentelijke organen 2017](https://vng.nl/files/vng/20170706-selectielijst-gemeenten-intergemeentelijke-organen-2017.pdf)
+- [Metadateren conform TMLO](https://www.gemmaonline.nl/index.php/Metadateren_van_zaakdossiers_conform_het_TMLO)
+- [DUTO Kwaliteitseisen](https://wiki.nationaalarchief.nl/pagina/DUTO:Kwaliteitseisen)
```

Comment:
deze wilde ik toevoegen ipv commenten, maar dat lukt me niet.

## Common Ground ##
De nieuwe Zaak DMS API die we opstellen moet bruikbaar zijn binnen de architectuur van Common Ground, zie 
- commonground.pleio.nl